docs/examples.md

```diff
@@ -508,20 +511,30 @@ import jenkins.automation.builders.BaseJobBuilder
 import jenkins.automation.utils.GhUtils
 
 new BaseJobBuilder(
-    name: "GHE_PR_builder",
-    description: "Does GHE PR building",
+    name: 'GHE_PR_builder',
+    description: 'Does GHE PR building from internal GHE',
 ).build(this).with {
-    GhUtils.ghPrWatcher(delegate,
-                        'orgname/reponame',
-                        'github.organization.tld',
-                        '<unique github credential UUID>',
-                        true
+    GhUtils.ghPrWatcher(
+        delegate,
+        [
+            ghProject: 'cfpb/reponame',
+            ghHostname: 'github.org.tld',
+            ghAuthId: '<GitHub user UUID>',
```

Comment:
I'm finding it's pretty hard to describe what this ID is, but in an accurate and concise way. Isn't this more the ID of the auth configured for the ghprb plugin? -- that bit under `<githubAuth>` in `org.jenkinsci.plugins.ghprb.GhprbTrigger.xml`? But that's pretty hard to say in the space allotted here. Any ideas for coming up with something a little more descriptive?

Reply:
It's the 'GitHub Pull Request Builder credential ID.' Seems wordy, but perhaps clearer.